ws-nextjs-app/cypress.config.ts

```diff
@@ -0,0 +1,41 @@
+/* eslint-disable no-param-reassign */
+import { defineConfig } from 'cypress';
+
+export default defineConfig({
+  e2e: {
+    setupNodeEvents(_on, config) {
+      if (!config.env.APP_ENV) {
+        config.env.APP_ENV = 'local';
+      }
+      const appEnv = config.env.APP_ENV;
+      const env = config.env[appEnv];
+
+      config.baseUrl = env.baseUrl;
+
+      // Debugging console logs to see running config
+      /* eslint-disable no-console */
+      console.log('\n\n\n\n\n');
+      console.log('Cypress running config:');
+      // console.log('SMOKE:', config.env.SMOKE);
+      console.log('APP_ENV:', config.env.APP_ENV);
+      console.log('Base URL:', config.baseUrl);
+      // console.log('UK:', config.env.UK);
+      console.log('\n\n\n\n\n');
+      /* eslint-enable no-console */
+
+      return config;
+    },
+    env: {
+      isNext: true,
```

Comment:
Nitpick but maybe we should call this `isNextJs`, just to be completely explicit?